### PR TITLE
Fix wrong jmx protocol and soap project properties for maven

### PR DESCRIPTION
**Pull request by mhpnguyen**
- Fix wrong jmx url in soapUI projects.
- Fix soapUI project properties for maven plugin which points to new embedded wildfly.

@danielrf123 @sailajaa @TabassumJafri : Can you please review this?

### CODE REVIEW

File: Product/SoapUI_Test/RegressionSuite/Standard/EntityDocQuerySecuredInterfaceTest/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     </parent>
 
     <artifactId>EntityDocQuerySecuredInterfaceTest</artifactId>
-    <name>Maven 3 soapUI EntityDocQuerySecuredInterfaceTest</name>
+    <name>Maven 3 soapUI EntityDocQuerySecuredInterfaceTest-Standard</name>
```
Comment (sailajaa) [inline Product/SoapUI_Test/RegressionSuite/Standard/EntityDocQuerySecuredInterfaceTest/pom.xml:10]: Not sure why this project renamed to "...-Standard"

Comment (mhpnguyen) [inline Product/SoapUI_Test/RegressionSuite/Standard/EntityDocQuerySecuredInterfaceTest/pom.xml:10]: This is for easy read when looking at failed nightly job. This won't affect soapUI project

File: Product/SoapUI_Test/RegressionSuite/Standard/XDRAsyncRequestInbound-Tests/pom.xml [hunk 2]
```diff
@@ -35,6 +35,9 @@
                     </soapuiProperties>
                     <testSuite>${test.suite}</testSuite>
                     <testCase>${test.case}</testCase>
+                    <projectProperties>
+                       <value>GatewayPropDir=${project.parent.parent.parent.basedir}/${wildfly.properties.dir}</value>
```
Comment (danielrf123) [inline Product/SoapUI_Test/RegressionSuite/Standard/XDRAsyncRequestInbound-Tests/pom.xml:39]: Where is the wildfly.properties.dir value set?

Comment (mhpnguyen) [inline Product/SoapUI_Test/RegressionSuite/Standard/XDRAsyncRequestInbound-Tests/pom.xml:39]: it locates under soapUI project level

**Comment by danielrf123** [comment]
Endorse for merge.

**Comment by sailajaa** [comment]
I reviewed and good with PR.

**Comment by TabassumJafri** [comment]
Endorsing for merge without further comments.

**Pull request merged**